qemu/tests/cfg/fio_windows.cfg

```diff
@@ -19,6 +19,6 @@
         install_cmd = 'msiexec /a DRIVE:\fio-x64.msi /qn TARGETDIR="${install_path}"'
     #config_cmd = 'setx -m path "%PATH%;${install_path}\fio;"'
     fio_cmd = '"${install_path}\fio\fio.exe" --rw=randrw --bs=4k --iodepth=4 --direct=1'
-    fio_cmd += ' --filename=%s --name=fiotest_%s --ioengine=windowsaio --thread --group_reporting'
-    fio_cmd += ' --numjobs=4 --size=1G --runtime=${cmd_timeout} > %s'
+    fio_cmd += ' --filename=${fio_file_name} --name=fiotest --ioengine=windowsaio --thread --group_reporting'
```

Comment:
the filename in fio cmd can be --filename=C\:\\fio_system_disk_test:I\:\\fio_data_disk_test ?

Reply:
No, ":" is the separator for filename, need "\" to translate

Reply:
But I think following file_name should be ok:
fio_file_name = "C\:\fio_system_disk_test:I\:\fio_data_disk_test"